src/dotnetTestExplorer.ts

```diff
@@ -111,28 +111,43 @@ export class DotnetTestExplorer implements TreeDataProvider<TestNode> {
         }
     }
 
+    private enableBuild(): string {
+        const dotnetBuildOption = Utility.getConfiguration().get<boolean>("build");
+        return dotnetBuildOption ? "" : "--no-build";
+    }
+
+    private enableRestore(): string {
+        const dotnetRestoreOption = Utility.getConfiguration().get<boolean>("restore");
+        return dotnetRestoreOption ? "" : "--no-restore";
+    }
+
+    private loadExclusion(name: string, value: string): string {
+        const exclusion = Utility.getConfiguration().get<string>(name);
+        return exclusion ? exclusion : value;
+    }
+
     private loadTestStrings(): Thenable<string[]> {
         this.evaluateTestDirectory();
 
-        let msBuildRootTestMsg = Utility.getConfiguration().get<string>("msbuildRootTestMsg");
-        msBuildRootTestMsg = msBuildRootTestMsg ? msBuildRootTestMsg : "The following Tests are available:";
+        const exclusions = [];
+
+        exclusions.push(this.loadExclusion("msbuildRootTestxUnitPrefix", "[xUnit.net"));

```

Comment:
Edited them for clearer depiction; 
![2017-08-22-07-55-001](https://user-images.githubusercontent.com/438196/29566395-572da4c0-870f-11e7-9b86-d3558ce25b37.png)